src/bilder/images/dagster/templates/docker-compose.yaml.j2

```diff
@@ -20,6 +20,7 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.http.routers.dagit.rule=Host(`${DAGSTER_HOSTNAME}`)"
+      - "traefik.http.routers.dagit.priority=1"
```

Comment:
This shouldn't be necessary if using the path prefix routing for the healthcheck container.